### PR TITLE
Added some suggestions for an improved structure

### PR DESCRIPTION
**Pull request by agemooij**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
-### What changes were proposed in this pull request?
+### What is the goal of this PR?
```
Comment (arjun-1) [inline .github/PULL_REQUEST_TEMPLATE.md]: Hmm, maybe it's personal but I feel there is some overlap between the 'goal' of a PR, and why the changes in the PR are needed. Quite generally, the goal would be to achieve some requirements. And these requirements are usually part of the why?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+
+
+### Links (JIRA, Notion, Miro)
```
Comment (arjun-1) [inline .github/PULL_REQUEST_TEMPLATE.md]: Would put this either in a smaller heading or at the bottom given that it doesn't need to be the most important thing for the reviewer to look at before starting the review.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (agemooij) [reply .github/PULL_REQUEST_TEMPLATE.md]: @vofque Then I would change the first header to be simply "Description". WDYT?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (vofque) [inline .github/PULL_REQUEST_TEMPLATE.md]: JIRA ticket should be linked automatically by extracting JIRA ticket key from PR name (I'm not sure that it works correctly from github side because I didn't see any links in my last PRs). And this naming convention should be probably documented in [contributors guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).

But there can also be some other related JIRA tickets, so yeah, makes sense.

**Review by vofque — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (agemooij) [reply .github/PULL_REQUEST_TEMPLATE.md]: I made the changes. WDYT?

Comment (vofque) [reply .github/PULL_REQUEST_TEMPLATE.md]: @agemooij "Description" sounds good to me.

**Review by arjun-1 — APPROVED** [review]
